Algorithm.Framework/QuantConnect.Algorithm.Framework.csproj

```diff
@@ -108,6 +108,7 @@
     <None Include="Portfolio\MeanVarianceOptimizationPortfolioConstructionModel.py">
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
+    <None Include="Portfolio\BlackLittermanPortfolioConstructionModel.py" />
```

Comment:
The `PreserveNewest` property is missing:

```
<None Include="Portfolio\BlackLittermanPortfolioConstructionModel.py">
  <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>	       
</None>
```

We are adding this to ensure that every time we compile, the newest version will be available.